packages/typespec-test/test/openai_modular/generated/typespec-ts/package.json

```diff
@@ -55,7 +55,9 @@
     "@azure-rest/core-client": "^1.1.3",
     "@azure/core-rest-pipeline": "^1.8.0",
     "@azure/logger": "^1.0.0",
-    "tslib": "^2.2.0"
+    "tslib": "^2.2.0",
+    "@azure/core-lro": "^2.5.3",
+    "@azure/abort-controller": "^1.0.0"
```

Comment:
why do we add this ? by the way,  Does the abortSignal really work now ?

Reply:
this is introduced by lro feature.